Sources/SmileID/Classes/Helpers/Colors.swift

```diff
@@ -1,7 +1,7 @@
 import SwiftUI
 
 extension Color {
```

Comment:
Do we need to make the extension public, or the init func only should be fine?

Reply:
Marking the extension public will expose every other method in the extension, but my thought is that we should only make the `init()` method accessible publicly.

Reply:
Thank you for the clarification